params/bamford_buzzard.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.5
```

Comment:
Hi @g1tKZ could you chose a different number here please, we've already seen a species with this parameter.